SEED-platform/seed
# 3254 investigate server with derived cols

#### Any background context you want to provide?
Derived columns are loaded twice during initial load on inventory_list_beta.

#### What's this PR do?
Refactor to prevent double loading and speed up load times.
removes duplicate requests:
- duplicate request for each derived column 
- duplicate get cycle by org id 
- duplicate properties filter


#### How should this be manually tested?
- generate 12 derived columns. ex: 12 identical columns defined as `pm_property_id / 100`
- monitor load times on develop
- monitor load times on this branch. Load times should be about 3 seconds faster.

#### What are the relevant tickets?
#3254 

#### Screenshots (if appropriate)
